packages/compass-indexes/src/components/create-index-modal/create-index-modal.tsx

```diff
@@ -55,11 +57,16 @@ function CreateIndexModal({
 
   useTrackOnChange(
     (track: TrackFunction) => {
+      const connectionInfo = connectionInfoAccess.getCurrentConnectionInfo();
```

Comment:
I noticed that in standard hooks you added connectionInfoAccess as a dependency but not here, maybe a miss?

Reply:
thank you!